### PR TITLE
2nd Draft of new readme

### PR DESCRIPTION
**Pull request by ROCKNROLLKID**
Continuing from https://github.com/MoscaDotTo/Winapp2/pull/269

I screwed up the last PR, so I head to re-open a new one. So here is draft 2 with variables and ExcludeKeys.

### CODE REVIEW

**Comment by TZocker** [comment]
PLS Add a section for:

| - the pipe symbol is used to separate file mask options.
*.* - deletes every file in that folder.
RECURSE - deletes subfolders and their contents
REMOVESELF - is the same as RECURSE but it deletes the folder as well.
* Deletes all files with the same extension (*.example) or all files with the same name (example.*)
; separates different files / filetypes in a filekey. 

also 

|*.exe|REMOVESELF 

greets

**Comment by ROCKNROLLKID** [comment]
Draft 3 is up.

**Comment by APMichael** [comment]
Thank you very much for the revised readme.

Something important: GitHub uses "Styling with Markdown". Therefore, you have to escape some special characters with a backslash \\. The special characters are for example the asterisk \* and the backslash \\ itself.

If you [view the whole file](https://github.com/MoscaDotTo/Winapp2/blob/e5e8d48f4c82ba4748b025f97bd19765c97a72ca/README.md) you will see that there are some asterisks and backslashes missing. That's why sometimes comments look wrong also.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ How to make your own entries:
 
 Winapp2 entries follow ABC ordering and we currently support all of the following code:
 
-[Name *] Name of the entry. Please include the space between the name and the *.
+[Name \*] Name of the entry. Please include the space between the name and the \*.
```
Comment (ROCKNROLLKID) [inline README.md]: This needs to be worded better, otherwise new people might think they need to include the \.

**Comment by APMichael** [comment]
The backslash isn't visible if people [view](https://github.com/MoscaDotTo/Winapp2/blob/1c4ea56075b0b4d4f666d224303426bc4f01d5a9/README.md) the readme.md on GitHub or if they just copy and paste it. It's only visible in the "source code" if they download the readme.md file.

If you want an "escape backslash free" file you have to change the readme.md from Markdown format to plain text format (= readme.txt).

**Comment by TZocker** [comment]
@APMichael 
Discussion about *.exe|removeself :
https://github.com/MoscaDotTo/Winapp2/pull/232

I think its a special example is a good I dear for it.

**Comment by APMichael** [comment]
If we stay on Markdown format, we can add some styling.

**Comment by ROCKNROLLKID** [comment]
If no one else has anything to add to this, I will go ahead and merge this.

**Pull request merged**